driver-core/src/main/com/mongodb/internal/operation/AsyncChangeStreamBatchCursor.java

```diff
@@ -211,7 +211,7 @@ private void resumeableOperation(final AsyncBlock asyncBlock, final SingleResult
 
     private void retryOperation(final AsyncBlock asyncBlock, final SingleResultCallback<List<T>> callback,
                                 final boolean tryNext) {
-        withAsyncReadConnection(binding, (source, t) -> {
+        withAsyncReadConnectionSource(binding, (source, t) -> {
```

Comment:
Renamed the method (it doesn't return a connection but a source), bringing it inline with the sync naming.